samples/MyEf.Hr/README.md

```diff
@@ -59,63 +59,23 @@ Endpoint | Description
-
-```
-└── MyEf.Hr               # Solution that references all underlying projects
-  └── Testing
-    └── MyEf.Hr.Test      # Unit and intra-integration tests
-  └── Tools
-    └── MyEf.Hr.CodeGen   # Entity and Reference Data code generation console
-    └── MyEf.Hr.Database  # Database code generation console
-  └── MyEf.Hr.Api         # API end-point and operations
-  └── MyEf.Hr.Business    # Core business logic components
-  └── MyEf.Hr.Common      # Common / shared components
-```
-
-_Note:_ Code generation should **not** be performed before updating the corresponding YAML files as described in the next sections. Otherwise, extraneous files will be generated that will then need to be manually removed.
-
-Also, any files that start with `Person` (being the demonstration entity) should be removed (deleted) from their respective projects as they are encountered. This then represents the baseline to build up the solution from.
-
-</br>
-
-## Railway-oriented programming
-
-_CoreEx_ version `3.0.0` introduced [monadic](https://en.wikipedia.org/wiki/Monad_(functional_programming)) error-handling, often referred to as [Railway-oriented programming](https://swlaschin.gitbooks.io/fsharpforfunandprofit/content/posts/recipe-part2.html). This is enabled via the key types of `Result` and `Result<T>`; please review the corresponding [documentation](https://github.com/Avanade/CoreEx/blob/main/src/CoreEx/Results/README.md) for more detail on purpose and usage. 
-
-The [`Result`](https://github.com/Avanade/CoreEx/blob/main/src/CoreEx/Results/Result.cs) and [`Result<T>`](https://github.com/Avanade/CoreEx/blob/main/src/CoreEx/Results/ResultT.cs) have been integrated into the code-generated output and is leveraged within the underlying validation. This is intended to simplify success and failure tracking, avoiding the need, and performance cost, in throwing resulting exceptions. 
-
 <br/>
 
-## Implementation steps
+## Core Implementation steps
 
 As described earlier, this sample will walk through the implementation in a number of logical steps:
 
-1. [Employee DB](./docs/Employee-DB.md) - creates the `Employee` database table and related entity framework capabilities.
-2. [Employee API](./docs/Employee-Api.md) - creates the `Employee` entities, API and related data access logic.
-3. [Employee Test](./docs/Employee-Test.md) - creates the `Employee` end-to-end integration tests to validate the API and database functionality.
-4. [Employee Search](./docs/Employee-Search.md) - adds the `Employee` search capability and tests.
-5. [Employee Terminate](./docs/Employee-Terminate.md) - adds the `Employee` termination capability and tests.
-6. [Employee Performance Review](./docs/Performance-Review.md) - adds the employee `PerformanceReview` capability end-to-end, from the the database, through the APIs and corresponding testing.
+1. [Solution Skeleton](./docs/Solution-Skeleton.md) - start by creating a VS solution from Beef template
```

Comment:
check all the markdown links are referencing the new file name with numbering prefix; e.g. `./docs/1-Solution-Skeleton.md`